src/Velo/Elaborator/Holey.idr

```diff
@@ -0,0 +1,109 @@
+module Velo.Elaborator.Holey
+
+import Data.DPair
+import Data.SnocList.Quantifiers
+import Decidable.Equality
+
+import Toolkit.Data.List.Pointwise
+import Toolkit.Data.List.Quantifiers
+import Toolkit.Data.List.Subset
+import Toolkit.DeBruijn.Context
+import Toolkit.DeBruijn.Variable
+
+import Velo.Core
+import Velo.Types
+import Velo.IR.Common
+import Velo.Elaborator.Common
+
+import Velo.IR.AST
+import Velo.IR.Holey
+
+%default total
+
+isVar : FileContext ->
```

Comment:
At somepoint *I* think it would be worth while to reintroduce the use of `DeBruijn` and it's kit for type-checking. But I can do that later.

Reply:
The problem with
```
decInfo
                  fc
                  (NotBound str)
                  (lookup str ctxt)
```
is that it forces us to make `types` runtime relevant even though that information is already available
in the `Item` stored in the context.

Reply:
mmhhh, good point.